### ISSUE TITLE
Create a refactoring test suite

### ISSUE THREAD

**Issue by axch** [edoddridge/MIM#6]
Goal: Provide a safety net for refactoring.

Plan: Several extremely short-time and low-resolution runs, checking that the answers are identical bit-for-bit with a previous version.  Can use the existing demo configurations as starting points.

Ideally:
- Cover all code paths.
- Execute as fast as possible.  I mean that: 100 ms would be ideal.

To cover the stochastic wind feature, will need to control the initial random seed, which is a separate issue.

**Comment by axch** [comment]
Careful inspection (carried out while producing PR #19) suggests the following list of features not yet covered by the refactoring test suite introduced therein:
- [ ] Sponge region(s)
- [ ] Non-empty depth file
- [ ] Sinusoidal wind
- [ ] Stochastic wind (blocked on Issue #8)
- [ ] Non-empty meridionalWindFile
- [ ] Non-zero initial U, V, and Eta
- [ ] Dumping wind files
- [ ] Non-zero freesurfFac (blocked on Issue #16, I think)

**Comment by axch** [comment]
Unassigning myself, since constructing the additional examples needed to get full coverage is not something I am doing immediately.
